docs/extras/blocking_nintendo.md

```diff
@@ -62,14 +62,14 @@ You can add a custom DNS to your WiFi connection that will block all communicati
     -----
 
     ### Using homebrew to set 90dns on all existing WiFi networks
-    1. Download [switch-90dns-setter](https://github.com/suchmememanyskill/switch-90dns-setter/releases) and put it in the switch folder.
+    1. Download [Switch-90dns-setter](https://github.com/suchmememanyskill/switch-90dns-setter/releases) and put it in the /switch folder.
```

Comment:
Minor suggestion, but since paths appear to all be in code blocks this should be as well.
```suggestion
    1. Download [Switch-90dns-setter](https://github.com/suchmememanyskill/switch-90dns-setter/releases) and put it in the `/switch` folder.
```